views/js/controller/creator/helpers/qtiTest.js

```diff
@@ -82,13 +82,29 @@ define(['lodash', 'i18n'], function(_, __){
             };
         },
 
+        /**
+         * Gives you a validator that check QTI id format of the test (it is different from the others...)
+         * @returns {Object} the validator
+         */
+        testidFormatValidator : function testidFormatValidator(){
+            var qtiTestIdPattern = /^\S+$/;
```

Comment:
I see... I believe the real bug actually is in the xsd but since we cannot wait for it to be officially modified, we would have to let this exception be.